src/run/parser/source_live_timer.rs

```diff
@@ -0,0 +1,106 @@
+//! Provides the parser for the SourceLiveTimer splits files.
+
+#![allow(non_snake_case)]
```

Comment:
That allows the whole module to ignore snake case. It's probably better to have it on the two structs instead.

Reply:
Makes sense, will change.